src/components/Dashboard/Dashboard.module.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../styles/color.scss';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 80vw;
```

Comment:
This is fully design decision (yours). But I feel like we could use more of the screen estate(more of its space)

![CleanShot 2023-10-12 at 11 26 33](https://github.com/KamilaSemotanova/tracker/assets/12488648/3f8bc022-2840-4040-b278-180659babe6d)

Reply:
Fixed, did not follow design correctly - media query will be added latter